app/buck2_client_ctx/src/subscribers/recorder.rs

```diff
@@ -450,6 +450,8 @@ mod imp {
                     _ => 0,
                 };
             self.command_end = Some(command);
+            // Prevent warning lint in OSS code.
+            #[allow(unused_variables)]
             let root = Path::to_owned(self.invocation_root_path.as_ref());
```

Comment:
This variable should be just moved inside cfg block.

Reply:
If it's moved into the cfg block, it results in far more unused triggers.

Reply:
I move it internal, but it involves adding a dead_code allow elsewhere.